spec/features/reservations_spec.rb

```diff
@@ -549,10 +561,8 @@
   context 'valid items on confirmation page' do
     before(:each) do
       empty_cart
-      select('10', from: 'items_per_page') # show more than 1 item (default)
       find('#items_per_form').submit_form!
```

Comment:
Damn, this line should be removed as well to save a request. This is what I thought I'd commented on last time, although I guess they were both extra. Sorry!